client/components/LearnerViewDirective.js

```diff
@@ -541,14 +545,14 @@ tie.directive('learnerView', [function() {
     `,
     controller: [
       '$scope', '$interval', '$timeout', 'SolutionHandlerService',
-      'QuestionDataService', 'LANGUAGE_PYTHON', 'FeedbackObjectFactory',
-      'ReinforcementObjectFactory', 'CodeStorageService',
-      'SECONDS_TO_MILLISECONDS', 'DEFAULT_AUTOSAVE_SECONDS',
-      'DISPLAY_AUTOSAVE_TEXT_SECONDS', 'SERVER_URL',
+      'QuestionDataService', 'LANGUAGE_PYTHON', 'ErrorTracebackObjectFactory',
+      'FeedbackObjectFactory', 'ReinforcementObjectFactory',
+      'CodeStorageService', 'SECONDS_TO_MILLISECONDS',
+      'DEFAULT_AUTOSAVE_SECONDS', 'DISPLAY_AUTOSAVE_TEXT_SECONDS', 'SERVER_URL',
       function(
           $scope, $interval, $timeout, SolutionHandlerService,
-          QuestionDataService, LANGUAGE_PYTHON, FeedbackObjectFactory,
-          ReinforcementObjectFactory, CodeStorageService,
+          QuestionDataService, LANGUAGE_PYTHON, ErrorTracebackObjectFactory,
+          FeedbackObjectFactory, ReinforcementObjectFactory, CodeStorageService,
```

Comment:
Nit: try to have the line breaks here match the above, so that it's easier to compare the two.